ddp-workspace/projects/ddp-prion/src/app/app-routing.module.ts

```diff
@@ -124,6 +125,11 @@ const routes: Routes = [
     component: LearnMoreComponent,
     canActivate: [IrbGuard]
   },
+  {
```

Comment:
This is technically part of a separate ticket, which makes it a little confusing to do the code review.  I'm mostly going to focus on the current ticket in this review.

Reply:
Done in f8fcc5a

Cut dummy private policy static page from prion project in this branch.
 
Left dummy private policy static page component in ddp-sdk.
Because cookie banner should open modal with private policy.

As agreed, private policy page will be implemented in your issue DDP-4519